### PR TITLE
Update Website Index and UserGuide

### PR DESCRIPTION
**Pull request by Pluiexo**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+StaffConnect (SF) is a **desktop app for managing contacts of Professors and Tutors, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, SF can get your contact management tasks done faster than traditional GUI apps.
```
Comment (tsulim) [inline docs/UserGuide.md]: Is it perhaps `SC` instead of `SF`?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
    * `list` : Lists all contacts.
 
-   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the StaffConnect contacts.
```
Comment (tsulim) [inline docs/UserGuide.md]: I noticed that the portions below only say `to the / in the contacts` whereas for this line here, it says `to the StaffConnect contacts`. Is there any particular reason for this?

**Review by tsulim — COMMENTED** [review]
Look good overall. 👍
Just some nits to fix/clarify.

File: docs/UserGuide.md (hunk 2 above)
Comment (Pluiexo) [reply docs/UserGuide.md]: I don't really understand what the  question is referring to here, Staffconnect contacts refers to the contacts list in the StaffConnect

File: docs/UserGuide.md (hunk 1 above)
Comment (Pluiexo) [reply docs/UserGuide.md]: Yep, accidental typo has been fix!

File: docs/UserGuide.md (hunk 2 above)
Comment (Pluiexo) [reply docs/UserGuide.md]: I have reworded the statement to ensure clarity now!

**Review by tsulim — APPROVED** [review]
LGTM! Thanks 👍

**Pull request merged**